### ISSUE TITLE
Can't get stack offset 0 from []

### ISSUE THREAD

**Issue by george-007** [KengoTODA/findbugs-slf4j#37]
I get the following error when running the maven plugin with ``mvn findbugs:findbugs``
```
findbugs-slf4j: 1.2.1
Apache Maven 3.3.3
both jdk1.8.0_60 and 7u76
```
```
[INFO] --- findbugs-maven-plugin:3.0.2:findbugs (default-cli) @ console ---
[INFO] Fork Value is true
     [java] The following errors occurred during analysis:
     [java]   Can't get stack offset 0 from [] @ 321 in ool.console.servlet.TinyMCEServlet.service : (Ljavax.servlet.http.HttpServletRequest;Ljavax.servlet.http.HttpServletResponse;)V
     [java]     java.lang.IllegalArgumentException: 0 is not a value stack offset
     [java]       At edu.umd.cs.findbugs.OpcodeStack.getStackItem(OpcodeStack.java:3100)
     [java]       At jp.skypencil.findbugs.slf4j.ManualMessageDetector.isInvokingGetMessage(ManualMessageDetector.java:59)
     [java]       At jp.skypencil.findbugs.slf4j.ManualMessageDetector.afterOpcode(ManualMessageDetector.java:50)
     [java]       At edu.umd.cs.findbugs.visitclass.DismantleBytecode.visit(DismantleBytecode.java:885)
     [java]       At edu.umd.cs.findbugs.visitclass.BetterVisitor.visitCode(BetterVisitor.java:218)
     [java]       At edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitCode(PreorderVisitor.java:235)
     [java]       At edu.umd.cs.findbugs.bcel.OpcodeStackDetector.visitCode(OpcodeStackDetector.java:63)
     [java]       At org.apache.bcel.classfile.Code.accept(Code.java:135)
     [java]       At edu.umd.cs.findbugs.visitclass.PreorderVisitor.doVisitMethod(PreorderVisitor.java:307)
     [java]       At edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitJavaClass(PreorderVisitor.java:395)
     [java]       At org.apache.bcel.classfile.JavaClass.accept(JavaClass.java:215)
     [java]       At edu.umd.cs.findbugs.BytecodeScanningDetector.visitClassContext(BytecodeScanningDetector.java:38)
     [java]       At jp.skypencil.findbugs.slf4j.parameter.AbstractDetectorForParameterArray.visitClassContext(AbstractDetectorForParameterArray.java:69)
     [java]       At edu.umd.cs.findbugs.DetectorToDetector2Adapter.visitClass(DetectorToDetector2Adapter.java:76)
     [java]       At edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1089)
     [java]       At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:283)
     [java]       At edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:402)
     [java]       At edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1200)
     [java] Exception in thread "main" java.lang.AssertionError: Unreachable
     [java] 	at jp.skypencil.findbugs.slf4j.parameter.ThrowableHandler.checkThrowable(ThrowableHandler.java:34)
     [java] 	at jp.skypencil.findbugs.slf4j.ManualMessageDetector.isInvokingGetMessage(ManualMessageDetector.java:59)
     [java] 	at jp.skypencil.findbugs.slf4j.ManualMessageDetector.afterOpcode(ManualMessageDetector.java:50)
     [java] 	at edu.umd.cs.findbugs.visitclass.DismantleBytecode.visit(DismantleBytecode.java:885)
     [java] 	at edu.umd.cs.findbugs.visitclass.BetterVisitor.visitCode(BetterVisitor.java:218)
     [java] 	at edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitCode(PreorderVisitor.java:235)
     [java] 	at edu.umd.cs.findbugs.bcel.OpcodeStackDetector.visitCode(OpcodeStackDetector.java:63)
     [java] 	at org.apache.bcel.classfile.Code.accept(Code.java:135)
     [java] 	at edu.umd.cs.findbugs.visitclass.PreorderVisitor.doVisitMethod(PreorderVisitor.java:307)
     [java] 	at edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitJavaClass(PreorderVisitor.java:395)
     [java] 	at org.apache.bcel.classfile.JavaClass.accept(JavaClass.java:215)
     [java] 	at edu.umd.cs.findbugs.BytecodeScanningDetector.visitClassContext(BytecodeScanningDetector.java:38)
     [java] 	at jp.skypencil.findbugs.slf4j.parameter.AbstractDetectorForParameterArray.visitClassContext(AbstractDetectorForParameterArray.java:69)
     [java] 	at edu.umd.cs.findbugs.DetectorToDetector2Adapter.visitClass(DetectorToDetector2Adapter.java:76)
     [java] 	at edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1089)
     [java] 	at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:283)
     [java] 	at edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:402)
     [java] 	at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1200)
```

**Comment by KengoTODA** [comment]
Thanks for your report. To reproduce this problem, I need more information especially about bytecode.

Could you share `ool.console.servlet.TinyMCEServlet.service` method with me? You can share it at here, or mail it to me.

**Comment by george-007** [comment]
Sent by email.

**Comment by KengoTODA** [comment]
Fix for this bug will be released as version `1.2.2`

**Comment by george-007** [comment]
Thanks a lot. I've tried the snapshot and it's working fine.
